portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -42,6 +42,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(convertGreetingsToJson());
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    /* Get the input from the form. */
+    String greeting = request.getParameter("text-input");
```

Comment:
What happens when the parameter does not exist?

Reply:
I think it adds an empty string to the list of strings. I could add an if statement to check if it's empty before adding.

Reply:
I think it returns a null string which won't cause an issue but I will add a check for null regardless. Thank you!